releng/build.Jenkinsfile

```diff
@@ -1,5 +1,9 @@
 pipeline {
   agent any
+  tools {
+    maven 'apache-maven-latest'
+    jdk 'adoptopenjdk-hotspot-jdk11-latest'
```

Comment:
Will we move to 11 with that? Shouldn’t we use 8?

Reply:
Yes - I was verifying what our old machine was setup as. The PR build was explicit, but the main build was not.